indexers/vector/PysparnnIndexer/tests/test_pysparnnindexer.py

```diff
@@ -102,7 +102,10 @@ def test_add_query_add_with_closing(indexer, features):
     indexer_query = PysparnnIndexer(metas=indexer.metas, metric='cosine')
     #  ValueError: Not possible query while indexing
     indexer_query.add(keys=list(range(0, 50)), vectors=features)
```

Comment:
this `add` is not so much an add as it is an `update`. It is expected to add new "non-existing" keys